content/docs/projects/README.md

```diff
@@ -15,6 +15,18 @@ These tools help with security, compliance and control.
 - [approver-policy](./approver-policy.md):
   a cert-manager **approver** that will automatically approve or deny
   certificate requests based on defined policy.
+- [csi-driver](./csi-driver.md):
+  csi-driver is a Container Storage Interface (CSI) driver plugin for Kubernetes
```

Comment:
nitpick (non-blocking): in other entries on this list we don't repeat the name of the project after the link. I think that stutters a bit; what do you think to removing it?

```suggestion
  a Container Storage Interface (CSI) driver plugin for Kubernetes
```

Reply:
Thanks, I agree. Let's remove the stutter